packages/cloud-cognitive/src/components/Datagrid/styles/useExpandedRow.scss

```
@@ -0,0 +1,13 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2020 - 2021
```

Can we update this to 2022?
```suggestion
* (c) Copyright IBM Corp. 2022
```